### DISCUSSION TITLE
Script lines on title not on texfield

### DISCUSSION THREAD

**Discussion by rice103** [arendst/Tasmota#10285] [Q&A] [answered]
Do you know if that it's a bug or it's normal?
![103179262-5f826580-488a-11eb-9477-3312c1402092](https://user-images.githubusercontent.com/35748699/103210013-bb89d000-4904-11eb-9b87-c344eb7ffe08.png)

**Comment by sfromis** [comment, marked as answer]
Due to recent fixes around scripting and browser, I'd suggest upgrading to latest dev version from http://ota.tasmota.com/tasmota/
Don't know if it will help, but worth trying.

**Reply by rice103** [reply]
I'm updated

**Reply by ascillato** [reply]
You have Tasmota 9.2.0
Please use the link above and update to latest 9.2.0.1
Thanks.

**Reply by rice103** [reply]
Thanks a lot it's working now
